types/sitemap-generator/index.d.ts

```diff
@@ -0,0 +1,47 @@
+// Type definitions for sitemap-generator 8.5
+// Project: https://github.com/lgraubner/sitemap-generator
+// Definitions by: grgr-dkrk <https://github.com/grgr-dkrk>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.9
```

Comment:
Which 3.9 features are required?

Reply:
Thanks.
I was using it only for `//@ts-expect-error` in the test file, thus I replaced it with`//$expect` and set the correct minimum version `2.8`.